### ISSUE TITLE
What do you do after ./buildall.sh?

### ISSUE THREAD

**Issue by dginovker** [rozniak/xfce-winxp-tc#295]
I'm on Fedora XFCE.

I ran `./chkdeps.sh` and installed all the missing dependencies.

Then I ran `./buildall.sh` and `cd`ed into `xptc` and installed all the `.rpm` files. There were no errors.

Then I rebooted and.. My desktop looks like the default XFCE.

**Comment by dginovker** [comment]
Ok looks like you go Settings > Appearance > select Windows XP style (Blue) > Toggle on `Set matching Xfwm4 theme if there is one`. Looks like I'm most of the way there, will add any more info as I find it

**Comment by dginovker** [comment]
Can you help me find some of the stuff to configure? I found

* `Mouse and Touchpad` > `Theme` > `Windows XP`

But my File Explorer/Task bar/etc are still very not Win XP theme-y

![image](https://github.com/rozniak/xfce-winxp-tc/assets/32943174/66833069-d6f1-442a-90b2-82914f63d6b0)

Thanks for all your work btw, the things that are working look amazing :D :heart:

**Comment by dginovker** [comment]
Update: Found `xfce-winxp-tc/packaging/build/shell/taskband/wintc-taskband` and `xfce-winxp-tc/packaging/build/shell/desktop/wintc-desktop` - Going to add those to my autostart

**Comment by rozniak** [comment]
Yeah the taskband works for now - `explorer` exists but is very WIP and only has barebones file browsing support atm.

**Comment by dginovker** [comment]
"WIP and only has barebones" you're too harsh on yourself :smile: It looks awesome!

This is gonna seem obvious, but also go `Appearance` > `Icons` > `Luna`

**Comment by rozniak** [comment]
I appreciate it :stuck_out_tongue: it's just there's quite a fair bit to be done on it before it can properly take the place of Thunar at least
